assets/queries/openAPI/operation_without_successful_http_status_code/query.rego

```diff
@@ -7,7 +7,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 	responses := doc.paths[n][oper].responses
 
-	count({x | response := responses[x]; regex.match(`^20[0124]$`, x) == true}) == 0
+	count({x | response := responses[x]; openapi_lib.content_allowed(oper, x); regex.match(`^20[0124]$`, x) == true}) == 0
```

Comment:
There is a reason to check if content is allowed here? Since this query is about checking if it has a response code, I think there is no reason to check content here...

Reply:
No content should be allowed for HEAD responses